client/core/account.go

```diff
@@ -2,11 +2,54 @@ package core
 
 import (
 	"encoding/hex"
+	"fmt"
 
 	"decred.org/dcrdex/client/db"
 	"github.com/decred/dcrd/dcrec/secp256k1/v3"
 )
 
+// AccountDisable is used to disable an account by given host and application
+// password.
+func (c *Core) AccountDisable(pw []byte, host string) error {
```

Comment:
Could you make a `TestAccountDisable` in `account_test.go`?

Reply:
surething

Reply:
donez

Reply:
Would like to suggest some changez.

https://github.com/amassarwi/dcrdex/pull/1/commits/e6fc9ca832bc0e45c6119004349023e1a2d5ae08

Reply:
Great improvement, thanks @JoeGruffins.
I will open another PR to adjust the other account tests to follow same approach.